papaya-core/src/main/java/pl/edu/agh/papaya/service/project/ProjectService.java

```diff
@@ -57,6 +61,17 @@ public boolean isUserInProject(Project project, String userId) {
                 .anyMatch(userId::equals);
     }
 
+    public boolean isUserInProject(Project project, User user) {
+        return isUserInProject(project, user.getId().toString());
+    }
+
+    public boolean hasUserAdminRights(Project project, User user) {
+        Optional<UserInProject> userInProject = getUserInProject(project, user);
+        return userInProject
```

Comment:
W ramach issue 55 powstało `Project#isAdmin`, to można pilnować, które się zmerge'uje pierwsze
I tak musimy być czujni z `ProjectService`, bo tam się pozmienia, jak #55 wejdzie